src/mongo/db/storage/wiredtiger/wiredtiger_kv_engine.cpp

```diff
@@ -2067,6 +2069,14 @@ Status WiredTigerKVEngine::hotBackup(OperationContext* opCtx, const std::string&
         fs::path destFile{std::get<1>(file)};
         auto fsize{std::get<2>(file)};
 
+        log() << "Hot backup copying {} bytes from '{}' to '{}'"_format(
+            fsize, srcFile.string(), destFile.string());
+        if (!fs::exists(srcFile)) {
+            log() << "Source file does not exist: {}"_format(srcFile.string());
+        } else {
+            log() << "Source file size is: {} bytes"_format(fs::file_size(srcFile));
+        }
+
```

Comment:
My idea was to show current size of the file. It may be different from the size reported in another message (that size is captured when file list is generated)